### ISSUE TITLE
Tests pass locally but fail on TravisCI

### ISSUE THREAD

**Issue by ospaarmann** [ospaarmann/exdgraph#5]
The tests pass locally but on TravisCI one test errors. To be more specific, the connection to the server fails.

```
04:29:15.948 [error] ExDgraph: Connection Failed
04:29:15.948 [error] ExDgraph.Protocol (#PID<0.1964.0>) failed to connect: ** (ErlangError) Erlang error: ExDgraph.Error
  1) MutationTest: failure on setup_all callback, test invalidated
     ** (MatchError) no match of right hand side value: {:error, {:callback_mode, :ok}}
     stacktrace:
       test/mutation_test.exs:19: MutationTest.__ex_unit_setup_all_0/1
       test/mutation_test.exs:1: MutationTest.__ex_unit__/2
?
  2) StarWarsSampleTest: failure on setup_all callback, test invalidated
     ** (exit) exited in: :gen_server.call(#PID<0.1948.0>, {:checkout, #Reference<0.0.1.9827>, true, 15000}, 5000)
         ** (EXIT) {:callback_mode, :ok}
     stacktrace:
       (db_connection) lib/db_connection/poolboy.ex:112: DBConnection.Poolboy.checkout/3
       (db_connection) lib/db_connection.ex:1518: DBConnection.checkout/3
       (db_connection) lib/db_connection.ex:852: DBConnection.run/3
       (ex_dgraph) lib/exdgraph/operation.ex:19: ExDgraph.Operation.operation/2
       test/starwars_sample_test.exs:16: StarWarsSampleTest.__ex_unit_setup_all_0/1
       test/starwars_sample_test.exs:1: StarWarsSampleTest.__ex_unit__/2
..
Finished in 0.2 seconds
6 tests, 2 failures, 2 invalid
```

See: https://travis-ci.org/ospaarmann/exdgraph/builds/350666336

**Comment by Fulnir** [comment]
I think the problem is GenStateMachine and the OTP version!
travis.yml
```
...
elixir:
  - 1.6
otp_release:
  - 20.0
...
```
I changed it, and make a pull request.

**Comment by ospaarmann** [comment]
Fixed by 63dc812b51ae2515ed6136c0a71ed3ae0ab3c41b 

Thanks @Fulnir
